src/components/shared/Breadcrumb/Breadcrumb.js

```diff
@@ -12,9 +13,17 @@ const BreadcrumbItem = ({ href, text }) => {
   return (
     <>
       {!!href ? (
-        <li>
-          <a href={href}>{text}</a>
-        </li>
+        href.startsWith('http') ? (
```

Comment:
I think that it follows the meaning more accurately:

```suggestion
        /^(?:[a-z]+:)?\/\//.test(href) ? (
```

Absolute: 
* `http://www.google.com`
* `ftp://www.google.com/folder`
* `//www.google.com`

Relative:
* `/folder`
* `folder`
* `folder.name`